code/modules/mod/modules/modules_ninja.dm

```diff
@@ -101,12 +101,12 @@
 		using the nanites already hosted in the wearer's suit to transmute into monomolecular shuriken. \
 		While these lack the intense bleeding edge of conventional throwing stars, \
 		they have been set to electrify fleeing targets; and branded with the Spider Clan symbol."
-	dispense_type = /obj/item/throwing_star/stamina/ninja
+	dispense_type = /obj/item/throwing_star/ninja
 	cooldown_time = 0.5 SECONDS
 
-/obj/item/throwing_star/stamina/ninja
+/obj/item/throwing_star/ninja
```

Comment:
I'm not going to block over this, but I do want to at least make sure you acknowledge my concern: Ninjas aren't allowed to murderbone and this is their only remotely non-lethal stopping tool since the players inside of nets can just break them. 

These "stamina" stars dealt 10 real brute damage upon hitting someone and then minor brute damage while embedded, but primarily they racked up stamina damage via embedding pain. This was a way for ninjas to take people down less lethally for the sake of capturing them alive or simply avoiding murder spree accusations since you can't always judge who's actively intending to attack and who's just gawking until it's too late because they have attacked you.

Reply:
My 2 points for this is:
- Nobody is going to be thinking of using the throwing star in situations where they want to be non-lethal, since its a throwing star and nothing about it indicates that it will be a non-lethal weapon.
- While Ninjas may not be allowed to murderbone, they can practically lethal anyone who isn't running away. If someone is captured in a net then critting them completes an objective, if someone is armed and willing to fight them then they are simply defending themselves, and if someone is standing there watching and not fleeing then that person is not attempting to preserve their own life making them a valid target.

I will move this out into another PR